src/main/java/vlad/petrovskyi/internetshop/controllers/user/LoginController.java

```diff
@@ -33,7 +33,7 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp)
             User user = authService.login(login, pass);
             HttpSession session = req.getSession();
             session.setAttribute("user_id", user.getId());
-            session.setAttribute("user_name", user.getName());
+            session.setAttribute("userName", user.getName());
```

Comment:
is it necessary?

Reply:
not that much - just wanted to add some difference between the general main page and a main page for specified user:) you can see the difference I wanted to add on the pictures I attached

<img width="1440" alt="Screenshot 2020-05-04 at 12 37 17" src="https://user-images.githubusercontent.com/62035381/80953401-1b76b800-8e04-11ea-88ef-5107658939d3.png">

<img width="1440" alt="Screenshot 2020-05-04 at 12 37 09" src="https://user-images.githubusercontent.com/62035381/80953380-10bc2300-8e04-11ea-8577-29edaf0eb910.png">